src/core/border_router/routing_manager.hpp

```diff
@@ -170,6 +170,7 @@ class RoutingManager : public InstanceLocator
         kMaxRaDelayTime              = 500, // The maximum delay of sending RA after receiving RS. In milliseconds.
         kRtrSolicitationInterval     = 4,   // The interval between Router Solicitations. In seconds.
         kMaxRtrSolicitationDelay     = 1,   // The maximum delay for initial solicitation. In seconds.
+        kMaxRoutingPolicyDelay       = 1,   // The maximum delay for routing policy evaluation. In seconds.
```

Comment:
1s is too quick ?

```suggestion
        kMaxRoutingPolicyDelay       = 3,   // The maximum delay for routing policy evaluation. In seconds.
```

Reply:
When the Routing Manager is started or all existing on-link prefixes are expired, we start
sending router solicitation in a random delay between 0 and 1 seconds. We are trying to use
the scale with it.